qubesadmin/device_protocol.py

```diff
@@ -284,10 +643,16 @@ class DeviceCategory(Enum):
     Mouse = ("u03**02", "p0902**")
     Printer = ("u07****",)
     Scanner = ("p0903**",)
-    # Multimedia = Audio, Video, Displays etc.
     Microphone = ("m******",)
-    Multimedia = ("u01****", "u0e****", "u06****", "u10****", "p03****",
-                  "p04****")
+    # Multimedia = Audio, Video, Displays etc.
+    Multimedia = (
+        "u01****",
+        "u0e****",
+        "u06****",
+        "u10****",
+        "p03****",
+        "p04****",
+    )
     Wireless = ("ue0****", "p0d****")
     Bluetooth = ("ue00101", "p0d11**")
     Mass_Data = ("b******", "u08****", "p01****")
```

Comment:
I'm not sure which PR is the right one to comment, but "Mass_Data" prefix in devices widget is IMO not very intuitive (but should be fine on the API level). Maybe better "Storage" or "Disk"? @marmarta any other ideas? This is how it looks: https://openqa.qubes-os.org/tests/116595#step/qui_widgets_devices/6

Reply:
Yes Mass_Data is not something useful, I would go with `Storage`.

Reply:
I agree, it is already changed ;)

Reply:
Sorry for the noise, I seen it after my comment.